go.mod

```diff
@@ -4,13 +4,13 @@ go 1.18
 
 require (
 	github.com/cenkalti/backoff v2.2.1+incompatible
```

Comment:
are all the module upgrades required?

Reply:
this is the result of running `go mod tidy`